api/src/org/apache/cloudstack/api/command/user/network/RestartNetworkCmd.java

```diff
@@ -57,6 +57,9 @@
     @Parameter(name = ApiConstants.CLEANUP, type = CommandType.BOOLEAN, required = false, description = "If cleanup old network elements")
     private Boolean cleanup;
 
+    @Parameter(name = ApiConstants.MAKEREDUNDANTE, type = CommandType.BOOLEAN, required = false, description = "Turn the network into a network with redundant routers.", since = "4.11.1")
```

Comment:
@rhtyd, Thanks for adding this functionality. I think there is a typo in ApiConstants name, E at the end. Would be great if we change the name of existing name to MAKE_RENDUNDANT, I just checked the usage is also very less.

Reply:
Thanks for spotting @nitin-maharana, will fix the typo